tapa_graphir/graphir/types/__init__.py

```diff
@@ -6,17 +6,17 @@
 RapidStream Contributor License Agreement.
 """
 
-from rapidstream.graphir.types.blackbox import BlackBox
-from rapidstream.graphir.types.commons import (
+from tapa_graphir.graphir.types.blackbox import BlackBox
+from tapa_graphir.graphir.types.commons import (
```

Comment:
`tapa_graphir.graphir` is a bit weird... suggest either keeping `rapidstream` as the top-level package name and make this `rapidstream.graphir`, or merging into `tapa`'s package hierarchy (I think the subpackages do not overlap with existing ones) and make this just `tapa.graphir`.